MekHQ/src/mekhq/campaign/stratcon/StratconRulesManager.java

```diff
@@ -0,0 +1,1080 @@
+        // data center - hostile - +5% scenario odds in track
+        // industrial center - allied - all scenarios here have THIS IS COMING OUT OF YOUR PAYCHECK modifier
+        // orbital defense - allied - sets 'no hostile aircraft' flag for track
+        // orbital defense - hostile - sets 'no allied aircraft' flag for track
+        // early warning system - allied - sets 'intercept allied base attacks' flag for track
+        // early warning system - hostile - sets 'intercept hostile base attacks' flag for track
+    }
+    
+    /**
+     * Process the deployment of a force to the given coordinates on the given track.
+     */
+    public static void processForceDeployment(StratconCoords coords, int forceID, Campaign campaign, StratconTrackState track) {
+        track.getRevealedCoords().add(coords);
+        StratconFacility facility = track.getFacility(coords);
+        if(facility != null) {
+            facility.setVisible(true);
+        }
+        
+        if(campaign.getLances().get(forceID).getRole() == AtBLanceRole.SCOUTING) {
+            for(int direction = 0; direction < 6; direction++) {
+                StratconCoords checkCoords = coords.translate(direction);
+                
+                facility = track.getFacility(checkCoords);
+                if(facility != null) {
+                    facility.setVisible(true);
+                }
+                
+                track.getRevealedCoords().add(coords.translate(direction));
+            }
+        }
+        
+        // the force may be located in other places on the track - clear it out
+        track.unassignForce(forceID);
+        track.assignForce(forceID, coords, campaign.getLocalDate());
+    }
+    
+    /**
+     * Worker function that processes the effects of deploying a reinforcement force to a scenario
+     */
+    public static boolean processReinforcementDeployment(ReinforcementEligibilityType reinforcementType, 
+            StratconCampaignState campaignState, StratconScenario scenario, Campaign campaign) {
+        // if the force is already deployed to the track, we're done
+        // if the force is a fight lance or we're using a support point
+        // if there is an SP to burn, burn it and we're done
+        // if there is a VP to burn, burn it and we're done
+        // now, roll 2d6 + lance commander tactics
+        // 9+ = deploy
+        // 6+ = deploy, if lance != fight, track scenario odds up
+        // 2+ = apply negative modifier to scenario
+        
+        if ((reinforcementType == ReinforcementEligibilityType.FightLance) ||
+                (reinforcementType == ReinforcementEligibilityType.SupportPoint)) {
+            
+            if (campaignState.getSupportPoints() > 0) {
+                campaignState.useSupportPoint();
+                return true;
+            } else if (campaignState.getVictoryPoints() > 0) {
+                campaignState.updateVictoryPoints(-1);
+                return true;
+            }
```

Comment:
If you changed `useSupportPoint` and `useVictoryPoint` you could make this simpler:
```java
    public boolean useSupportPoint() {
        if (supportPoints > 0) {
            supportPoints--;
            return true;
        }
        return false;
    }
```
Used here like:
```java
    if (campaignState.useSupportPoint() || campaignState.useVictoryPoint()) {
       return true;
    }     
```